nimare/meta/cbma/mkda.py

```diff
@@ -204,7 +206,19 @@ def _fit(self, dataset1, dataset2):
         if "ma_maps1" in self.inputs_.keys():
```

Comment:
since this code is repeated (and now contains work to use a memmap file), it would be more readable as a function such as `get_ma_maps`, which could be defined in `CBMAEstimator` I think. (it would have to take `self.memmap_filenames[index]` as an argument)

Reply:
Good point! Can do.